sdk/kubeflow/trainer/api/trainer_client.py

```diff
@@ -170,29 +183,63 @@ def train(
+
+                # Add resources per node to the Trainer.
+                if trainer.resources_per_node:
+                    trainer_crd.resources_per_node = utils.get_resources_per_node(
+                        trainer.resources_per_node
+                    )
 
-        # Add resources per node to the Trainer.
-        if trainer and trainer.resources_per_node:
-            trainer_crd.resources_per_node = utils.get_resources_per_node(
-                trainer.resources_per_node
-            )
+                # Add command and args to the Trainer.
+                trainer_crd.command = constants.DEFAULT_CUSTOM_COMMAND
+                # TODO: Support train function parameters.
+                trainer_crd.command, trainer_crd.args = (
+                    utils.get_entrypoint_using_train_func(
+                        runtime,
+                        trainer.func,
+                        trainer.func_args,
+                        trainer.pip_index_url,
+                        trainer.packages_to_install,
+                    )
+                )
 
-        # Add command and args to the Trainer if training function is set.
-        if trainer and trainer.func:
-
-            # TODO: Support train function parameters.
-            trainer_crd.command, trainer_crd.args = (
-                utils.get_entrypoint_using_train_func(
-                    runtime,
-                    trainer.func,
-                    trainer.func_args,
-                    trainer.pip_index_url,
-                    trainer.packages_to_install,
+            # If users choose to use a builtin trainer for post-training.
+            elif isinstance(trainer, types.BuiltinTrainer):
+                if trainer.config is None or not isinstance(
+                    trainer.config, types.TorchTuneConfig
+                ):
+                    raise ValueError(
+                        "The config is missing or not supported for the BuiltinTrainer. "
+                        "Currently, we only support `TorchTuneConfig` as parameter."
+                    )
+
+                # Add number of nodes to the Trainer.
+                if trainer.config.num_nodes:
+                    trainer_crd.num_nodes = trainer.config.num_nodes
+
+                # Add resources per node to the Trainer.
+                if trainer.config.resources_per_node:
+                    trainer_crd.resources_per_node = utils.get_resources_per_node(
+                        trainer.config.resources_per_node
+                    )
+
+                # Parse args in the TorchTuneConfig to the Trainer, preparing for the mutation of
+                # the torchtune config in the runtime plugin.
+                # Ref:https://github.com/kubeflow/trainer/tree/master/docs/proposals/2401-llm-trainer-v2
+                trainer_crd.command = constants.DEFAULT_TORCHTUNE_COMMAND
+                trainer_crd.args = utils.get_args_using_torchtune_config(trainer)
```

Comment:
Please can you move the command and args assignment to the helper function, so we will be consistent with
https://github.com/kubeflow/trainer/blob/d5ab621676524130271a121d4292a3ca851b5c3a/sdk/kubeflow/trainer/api/trainer_client.py#L203

```suggestion
                trainer_crd.command, trainer_crd.args = utils.get_args_using_torchtune_config(trainer)
```

Reply:
Sure!